packages/sentinel/src/api/index.ts

```diff
@@ -107,6 +107,12 @@ export class SentinelClient extends BaseApiClient {
     });
   }
 
+  public async deleteNotificationChannel(notification: NotificationResponse): Promise<string> {
```

Comment:
We have `ExternalUpdateSubscriberRequest` and `ExternalCreateSubscriberRequest` types. Let's create also a `ExternalDeleteSubscriberRequest` for the sake of consistency.

I've already had some problems with the sentinels client before in #94 because of something similar

Reply:
This PR is only introducing a delete *notification* endpoint. The delete subscriber endpoint only takes in the `subscriberId` as a parameter.